app/editor/src/features/admin/ingests/configurations/FrontPage.tsx

```diff
@@ -67,6 +67,7 @@ export const FrontPage: React.FC = (props) => {
           />
         </Col>
       </Row>
+      <FormikCheckbox label="Delete original files" name="configuration.deleteOriginal" />
```

Comment:
A way to control whether the original files are deleted.  This is required because the SFTP location slowly files up and it takes longer each time to parse the files.